src/Microsoft.ML.Mkl.Components/SymSgdClassificationTrainer.cs

```diff
@@ -48,6 +48,7 @@ namespace Microsoft.ML.Trainers
     /// | Is normalization required? | Yes |
     /// | Is caching required? | No |
     /// | Required NuGet in addition to Microsoft.ML |Microsoft.ML.Mkl.Components |
+    ///
     /// ### Training Algorithm Details
     /// The symbolic SGD is a classification algorithm that makes its predictions by finding a separating hyperplane.
     /// For instance, with feature values $f0, f1,..., f_{D-1}$, the prediction is given by determining what side of the hyperplane the point falls into.
```

Comment:
The actual scoring function is $\boldsymbol{w}^T \boldsymbol{x} + b$ but this description only mentions $\boldsymbol{x}$.